packages/components/src/components/@shared/_kol-table-stateless-mixin.scss

```diff
@@ -95,63 +96,63 @@
 				width: var(--a11y-min-size);
 			}
 
-			.icon {
+			&__selection-icon {
 				display: block;
 				inset: auto;
 				position: absolute;
 				z-index: 1;
 			}
 
-			input {
+			&__selection-input {
 				appearance: none;
 				border-style: solid;
 				cursor: pointer;
 				margin: 0;
-			}
 
-			input:before {
-				content: '';
-			}
+				&:before {
+					content: '';
+				}
 
-			input[type='checkbox'] {
-				position: relative;
-				display: flex;
-				height: calc(var(--a11y-min-size) / 2);
-				width: calc(var(--a11y-min-size) / 2);
-				align-items: center;
-				justify-content: center;
-				background-color: rgb(255, 255, 255);
-				border-width: rem(2);
-				line-height: 1.5;
-				transition: all 0.5s ease 0s;
-			}
+				&[type='checkbox'] {
+					position: relative;
+					display: flex;
+					height: calc(var(--a11y-min-size) / 2);
+					width: calc(var(--a11y-min-size) / 2);
+					align-items: center;
+					justify-content: center;
+					background-color: rgb(255, 255, 255);
+					border-width: rem(2);
+					line-height: 1.5;
+					transition: all 0.5s ease 0s;
+				}
 
-			input[type='radio'] {
-				display: flex;
-				border-width: rem(2);
-				border-radius: 100%;
-				height: 1.5em;
-				min-height: 1.5em;
-				min-width: 1.5em;
-				padding: 0;
-				width: 1.5em;
-			}
+				&[type='radio'] {
+					display: flex;
+					border-width: rem(2);
+					border-radius: 100%;
+					height: 1.5em;
+					min-height: 1.5em;
+					min-width: 1.5em;
+					padding: 0;
+					width: 1.5em;
+				}
 
-			input[type='radio']:before {
-				border-radius: 100%;
-				margin: auto;
-				height: calc(1.5em / 2);
-				width: calc(1.5em / 2);
-			}
+				&[type='radio']:before {
+					border-radius: 100%;
+					margin: auto;
+					height: calc(1.5em / 2);
+					width: calc(1.5em / 2);
+				}
 
-			input[type='radio']:checked:before {
-				background-color: #000;
-			}
+				&[type='radio']:checked:before {
+					background-color: #000;
+				}
 
-			@media (forced-colors: active) {
-				input[type='radio']:checked:before {
-					/* Give it a visible background in forced colors mode */
-					background-color: selectedItem !important;
+				@media (forced-colors: active) {
```

Comment:
Mit Sass können wir `@media` in beide Richtungen verschachteln. Bitte direkt innerhalb vom `[type='radio']:checked:before`-Block einfügen.